### PR TITLE
Issues/213 404 handler

### PR DESCRIPTION
**Pull request by marcemarc**
Added some info on legacy not found handlers - pointed people in direction of IContentFinders

### CODE REVIEW

File: Reference/Config/404handlers/index.md [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+    <NotFoundHandlers>
+      <notFound assembly="umbraco" type="SearchForAlias" />
+      <notFound assembly="umbraco" type="SearchForTemplate"/>
+      <notFound assembly="umbraco" type="SearchForProfile"/>
+      <notFound assembly="umbraco" type="handle404"/>
+    </NotFoundHandlers>
+
+The concept of *NotFoundHandlers* has been replaced by *IContentFinders* - [Implementing an IContentFinder](../../routing/request-pipeline/IContentFinder)
+
+Custom 404's are now handled by creating a custom IContentFinder registered as a *ContentLastChanceFinder* - [Using an IContentFinder for Custom 404s](../../routing/request-pipeline/IContentFinder#notfoundhandlers)
```
Comment (dampee) [inline Reference/Config/404handlers/index.md]: You are suggesting there are multiple 404's possible with ContentLastChange, while this is the only and last one.  Every content finder can be a custom 404.

**Comment by dampee** [comment]
Only one small remark.  For the rest it is good to go and should have been in the docs for ages.  ;-)

**Comment by marcemarc** [comment]
@dampee tweaked, is that more technically correct now... ?

**Comment by dampee** [comment]
Thanks a lot Marc.

**Pull request merged**